x-pack/plugins/ml/server/models/data_recognizer/modules/sample_data_weblogs/ml/count_by_response_code.json

```diff
@@ -0,0 +1,37 @@
+{
+  "groups": ["kibana_sample_logs", "kibana_sample_data"],
+  "description": "Kibana sample web logs data : find anomalies in the event count by HTTP response code ",
+  "analysis_config": {
+    "bucket_span": "1h",
+    "detectors": [
+      {
+        "detector_description": "Event count by response code",
+        "function": "count",
+        "partition_field_name": "response.keyword"
+      }
+    ],
+    "influencers": ["clientip"]
```

Comment:
The partitioning field `response.keyword` should be added as an influencer here, as it will provide extra useful information, such as being able to view by response code in the Anomaly Explorer swimlane.